Gems/AtomLyIntegration/EditorModeFeedback/Code/Source/EditorModeFeedbackSystemComponent.cpp

```diff
@@ -75,5 +81,35 @@ namespace AZ
         {
             return m_registeryEnabled;
         }
+
+        void EditorModeFeedbackSystemComponent::OnStartPlayInEditorBegin()
+        {
+            SetEnableRender(false);
+        }
+
+        void EditorModeFeedbackSystemComponent::OnStopPlayInEditor()
+        {
+            SetEnableRender(true);
+        }
+
+        void EditorModeFeedbackSystemComponent::SetEnableRender(bool enableRender)
+        {
+            auto sceneSystem = AzFramework::SceneSystemInterface::Get();
+            if (sceneSystem)
```

Comment:
nit: Could move this assignment inside the if to limit scope

```
if (auto* sceneSystem = AzFramework::SceneSystemInterface::Get())
// or
if (auto* sceneSystem = AzFramework::SceneSystemInterface::Get();
    sceneSystem)
```

Depending on your preference